lib.rs

```diff
@@ -464,6 +463,78 @@ impl ThreadPool {
     }
 }
 
+impl Clone for ThreadPool {
+    /// Cloning a pool will create a new handle to the pool.
+    /// The behavior is similar to [Arc](https://doc.rust-lang.org/stable/std/sync/struct.Arc.html).
+    ///
+    /// We could for example [`join`](#method.join) a pool from multiple
+    /// threads or submit jobs from multiple threads concurrently.
+    ///
+    /// ```
```

Comment:
Sorry for the delay, I'm thinking if there's a simpler example we can put here. I know you're spawning threads so you can show that the `ThreadPool` can be cloned, but do we need much of the other code here?